### ISSUE TITLE
dynamic key names possible in root?

### ISSUE THREAD

**Issue by lithiumlab** [seperman/redisworks#1]
Is is possible to use dynamic keys?
```
		val = '12345'
		root.['test_'+val] = 'HELLO WORLD'
		print root.test_12345

```

**Comment by seperman** [comment]
@lithiumlab This is a bug. It should accept `root['test_'+val] = 'HELLO WORLD'` (without the dot).
I will look into it asap.

**Comment by seperman** [comment]
The root of this issue is in DotObject which I just fixed there.
https://github.com/seperman/dotobject/issues/2

**Comment by seperman** [comment]
I just published 0.2.7 and this issue is fixed.

Now you can: `root['test_'+val] = 'HELLO WORLD'`

and even if you do

```
root['blah'] = 'foo'
root['blah.ding'] = 'foo too'
```
later you can access it as

```
root.blah
root.blah.ding
```
